KillTeam/Commands/DeleteTeamCommand.cs

```diff
@@ -0,0 +1,12 @@
+namespace KillTeam.Commands
+{
+    public class DeleteTeamCommand
+    {
+        public string TeamId { get; }
+
+        public DeleteTeamCommand(string teamId)
+        {
+            TeamId = teamId;
+        }
+    }
+}
```

Comment:
If the command is just a thin wrapper class, does C# have a good struct equivalent we could use instead of a full class?

Reply:
Yes we can use `struct` in C# which by and large would probably work for the Commands. The only downside is that they are value types so in some scenarios you need to `...(ref myCommand)` to maintain the reference. 

An example of this is where a CommandHandler persists a new entity, and you want store the new Id in the Command you pass in to access it after the `Handle` method has completed. It's not a reason not, just something to be aware of when you do.

Reply:
Yep, we have to stick with a class here.